sdk/cosmos/azure-cosmos/azure/cosmos/database.py

```diff
@@ -232,9 +232,7 @@ def create_container(
                 UserWarning,
             )
             request_options["populateQueryMetrics"] = populate_query_metrics
-        if offer_throughput is not None:
```

Comment:
you should keep the checks for not None and do _throughput_settings() if it's not None

Reply:
nvm im silly, didn't see the check in the _throughput_settings method